backend/market/serializers.py

```diff
@@ -50,46 +61,41 @@ def get_image_url(self, obj):
             return image.url
 
     class Meta:
-        model = SubletImage
+        model = ItemImage
         fields = ["id", "image_url"]
 
 
-# complex sublet serializer for use in C/U/D + getting info about a singular sublet
-class SubletSerializer(serializers.ModelSerializer):
-    # amenities = AmenitySerializer(many=True, required=False)
-    # images = SubletImageURLSerializer(many=True, required=False)
-    amenities = serializers.PrimaryKeyRelatedField(
-        many=True, queryset=Amenity.objects.all(), required=False
+# complex item serializer for use in C/U/D + getting info about a singular tag
+class ItemSerializer(serializers.ModelSerializer):
+    # amenities = ItemSerializer(many=True, required=False)
+    images = ItemImageURLSerializer(many=True, required=False)
+
+    tags = serializers.PrimaryKeyRelatedField(many=True, queryset=Tag.objects.all(), required=False)
+    category = serializers.PrimaryKeyRelatedField(queryset=Category.objects.all(), required=True)
+    seller = serializers.PrimaryKeyRelatedField(queryset=User.objects.all(), required=False)
+    favorites = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=User.objects.all(), required=False
     )
```

Comment:
Yes removing those fields doesn't change functionality in this case. The Item model determines whether a field is required or not.

Reply:
Just to clarify my understanding, we don't want to return seller info on GET requests since it's not relevant until a buyer makes an offer (which should be differentiated from a general GET request).
Is this correct?